Source/WebGPU/WebGPU/CommandEncoder.swift

```diff
@@ -533,6 +533,7 @@ extension WebGPU.CommandEncoder {
                 mtlDescriptor.sampleBufferAttachments[0].endOfVertexSampleIndex = Int(descriptor.timestampWrites.pointee.endOfPassWriteIndex)
                 mtlDescriptor.sampleBufferAttachments[0].startOfFragmentSampleIndex = Int(descriptor.timestampWrites.pointee.endOfPassWriteIndex)
                 mtlDescriptor.sampleBufferAttachments[0].endOfFragmentSampleIndex = Int(descriptor.timestampWrites.pointee.endOfPassWriteIndex)
+                m_device.trackTimestampsBuffer(m_commandBuffer, counterSampleBuffer);
```

Comment:
I need to build this, but didn't want to forget about the corresponding swift changes @nmahendru